README.md

```diff
@@ -45,34 +45,27 @@ declared in the `pyproject.toml` file.
 ## Testing
 
 Running tests or examples requires a local git clone or unpacked source distribution.
+Install test suite dependencies with `pip install '.[test]'`.
 
 Some tests can optionally output figures or extended diagnostics when run locally.
-Testing locally requires a local git clone or source distribution of PyDRex.
 Check the `tests/README.md` file for details.
 
 Further examples that demonstrate how PyDRex can be used within geodynamic
-simulations are provided in the `examples` folder.
+simulations are provided in subfolders of the `examples` folder.
 They have their own README file as well.
 
 ## Building offline documentation
 
-The documentation can be built offline using [pdoc](https://github.com/mitmproxy/pdoc),
-with the command:
+The documentation can be built offline from a git clone or source distribution.
+Install documentation builder dependencies with `pip install '.[doc]'`.
 
-    pdoc -t docs/template -o html --math pydrex tests
-
-which will output the html documentation into a folder called `html`.
+Run the make command `make html` which will output the documentation,
+including the generated API reference, into a folder called `html`.
```

Comment:
Run `make html` from the terminal to generate PyDRex's documentation (available in the `html` directory), including the API reference.